### ISSUE TITLE
iOS 17 + Google app + Video Autoplay feature = Blackout

### ISSUE THREAD

**Issue by kawanet** [Modernizr/Modernizr#2735]
**Type of bug**

- [✔] Modernizr error

**Describe the bug**

Google app https://apps.apple.com/app/id284815942 goes blackout screen when opening a page which contains Modernizr "Video autoplay" feature enabled.

- iOS 17.1.2 + Google app = NG
- iOS 17.1.2 + Safari = NG
- iOS 17.1.2 + Chrome = OK
- iOS 16 + Google app = OK

**How to Reproduce**

Build the Modernizr with "Video autoplay" feature enabled: https://modernizr.com/download?-videoautoplay-dontmin
Write the following minimum HTML file to reproduce:

```
<script src="modernizr-custom.js"></script>
```

Access it with iOS Google app https://apps.apple.com/app/id284815942 on iPhone device, iOS 17.

**Expected behavior**

- No blackout as other browsers.

**Comment by Pastafarian** [comment]
Yup, just ran into this myself. Upgraded the version of Modernizr as I was upgrading jQuery anyway. This one bit us in the ass as it only showed up in mobile testing in our app at the last moment.
